core/components/patterns/table/TableWithHeader/tableWithHeader.story.jsx

```diff
@@ -674,10 +674,12 @@ import './style.css';
         loading
       } = this.state;
       const totalPages = getTotalPages(totalRecords, pageSize);
-  
+
+      const classNames = showVerticalFilters ? 'Table-verticalFilter' : 'w-100';
```

Comment:
Please use `classNames` utility to get dynamic class-name strings.

Reply:
We can not use `classNames` in custom code. [line22](https://github.com/innovaccer/design-system/blob/master/core/components/patterns/datePicker/datePickerWithPresets.story.tsx)